### ISSUE TITLE
Start casting to a group of speakers?

### ISSUE THREAD

**Issue by redskinwashingtons** [tprelog/homeassistant-gmusic_player#5]
I have 3 Google Home devices in my house, and usually I just cast my music to a group I made that has those 3 devices in it. From gmusic in HA I seem to only be able to cast to a specific media_player/device, not the group. My group ("speakers") correctly shows up in my device list (media_player.speakers) and I'm able to use it with Mini Media Player as well. 

Casting doesn't throw an error, and it shows as playing, but nothing happens then.

**Comment by tprelog** [comment]
I have tried to get this working for a while now myself. Even outside of this component I am not able to play music this way to a group of cast devices so I don't think it will be possible in this form.

For example, playback from gmusicproxy will fail to group:
- 192.0.1.19:9999 is example IP:PORT of gproxy
- media_player.speaker_group is group containing cast devices

From **Developer Tools --> Services --> `media_player.play_media`**
```
{
  "entity_id": "media_player.speaker_group",
  "media_content_id":"http://192.0.1.19:9999/get_song?id=Tscf4m2vahago3g2ix434ehck6u",
  "media_content_type":"music"
}
```
In other words, because the above example does not work, it will not work here because the gmusic_player itself is just a controller that basically looks up the `media_content_id` of a song and uses the `media_player.play_media` service call to the selected `entity_id` from the device in the list.

**Comment by tprelog** [comment]
> My group ("speakers") correctly shows up in my device list (media_player.speakers)

 Did you create this group in HA or using the "Google Home" app?

I have previously been trying to create my cast groups using HA and playback never seems to work.

In an unrelated event, I created a cast group using the [Google Home](https://play.google.com/store/apps/details?id=com.google.android.apps.chromecast.app&hl=en_US) app. I was surprised to see my new cast group automagically added to HA. Anyways, gmusic_player seems to function as expected when using the new cast group created with "Google Home".
